sdks/python/apache_beam/testing/benchmarks/inference/pytorch_image_classification_benchmarks.py

```diff
@@ -0,0 +1,63 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# pytype: skip-file
+
+import logging
+
+from apache_beam.examples.inference import pytorch_image_classification
+from apache_beam.testing.load_tests.load_test import LoadTest
+from torchvision import models
+
+_PERF_TEST_MODELS = ['resnet50', 'resnet101', 'resnet152']
+_PRETRAINED_MODEL_MODULE = 'torchvision.models'
+
+
+class PytorchVisionBenchmarkTest(LoadTest):
+  def __init__(self):
+    # TODO (anandinguva): make get_namespace() method in RunInference static
```

Comment:
Do you mean `get_metrics_namespace()`? Let's add a GH issue to this.

Reply:
Yep. Right now, I would need to instantiate a ModelHandler to get the namespace. For instantiation, I would need few parameters which are not necessarily available here in this .py file. 

Making `get_metrics_namespace()` static would be ideal solution in this case

Reply:
Agreed - could you update to `get_metrics_namespace` and link to an issue in the TODO instead of just your username?